hack/ci-e2e-test.sh

```diff
@@ -2,6 +2,19 @@
 
 set -e
 
+function cleanup {
```

Comment:
This has to come first as this is a script, otherwise if it fails midways it will just exit as it didn't read the trap code yet